internal/certdb/validation.go

```diff
@@ -0,0 +1,48 @@
+package certdb
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+)
+
+func ValidateCertificateRequest(csrString string) error {
```

Comment:
Please add docstrings on the public functions in this file too.

Reply:
Done